torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -789,6 +789,36 @@ def test_backend_group(self):
         def test_backend_full_group(self):
             self._test_group_override_backend(self._init_full_group_test)
 
+        @skip_if_lt_x_gpu(int(os.environ["WORLD_SIZE"]))
+        def test_new_default_subgroups_nccl(self):
+            world_size = dist.get_world_size()
+            cur_subgroup, subgroups = dist.new_subgroups()
```

Comment:
Would be good to test passing in non-default group for this part of the test and below as well

Reply:
Non-default group is not accepted now.